packages/design-tokens/package.json

```diff
@@ -17,37 +17,46 @@
     "js",
     "dist"
   ],
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.cjs",
+  "module": "dist/esm/index.mjs",
+  "types": "dist/types/index.d.ts",
   "private": false,
   "license": "MIT",
-  "sideEffects": false,
+  "sideEffects": true,
   "scripts": {
     "lint:ts": "tsc --noEmit",
     "test": "FORCE_COLOR=1 jest",
     "test:ci": "pnpm test -- --ci",
     "build": "pnpm clean && pnpm build:json && pnpm build:less && pnpm build:sass && pnpm build:ts",
     "build:json": "tsx ./bin/buildCssVarTokens.ts",
-    "build:ts": "tsc --project tsconfig.dist.json",
+    "build:ts": "pnpm package-bundler build",
     "build:less": "json-to-flat-sass './tokens/*.json' 'less' --extension 'less' --caseType 'kebab' && prettier less/* --write",
     "build:sass": "json-to-flat-sass './tokens/*.json' 'sass' --extension 'scss' --caseType 'kebab' && prettier sass/* --write",
+    "lint:scss": "pnpm stylelint './src/**/*.scss' --config .stylelintrc-scss.json",
+    "lint:styles": "pnpm lint:css && pnpm lint:scss",
```

Comment:
I can't seem to see a `lint:css`

Reply:
Oops, leftover from an experiment with getting the packages to individually lint